test/Language/Github/Actions/WorkflowTest.hs

```diff
@@ -50,13 +49,13 @@ test_goldenWorkflowFromYaml = do
               putStrLn $ "roundtrip " <> takeBaseName testYamlFilePath
               eitherWorkflow <- Yaml.decodeFileEither @Workflow testYamlFilePath
               either
-                (BS.writeFile outputFilePath >> (\e -> fail $ "YAML decoding failed: " ++ show e))
+                (writeFile outputFilePath >> (\e -> fail $ "YAML decoding failed: " ++ show e))
```

Comment:
This one is probably an improvement because you get a `String` from the yaml lib and have to walk the whole thing anyway.